__tests__/app.test.js

```diff
@@ -153,4 +150,34 @@ describe("app/articles", () => {
         });
     });
   });
+  describe("POST/api/articles/:article_id/comments", () => {
+    test("200: returns the comment that was just posted to the correct article_id given", () => {
+      return request(app)
+        .post("/api/articles/3/comments")
+        .send({ username: "butter_bridge", body: "who is Mitch?" })
+        .expect("Content-Type", "application/json; charset=utf-8")
+        .expect(201)
+        .then(({ body }) => {
+          const addedComment = body.newComment[0];
+          expect(addedComment).toMatchObject({
+            author: "butter_bridge",
+            body: "who is Mitch?",
+            article_id: 3,
+            comment_id: expect.any(Number),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
```

Comment:
add another 201 test to test that when adding a comment it ignores any properties that aren't needed!